src/scripts/components/header/SearchInput.js

```diff
@@ -27,6 +27,20 @@ class SearchInput extends React.Component {
         });
       }
     }
+
+    // This checks to see if the user is typing Arabic
+    // and adjusts the text-align. 
+    var classes = document.getElementsByClassName("SearchMeat");
```

Comment:
Hmmm, this doesn't follow the es6 classes. It must be contained within an instance method and not on it's own. I'd suggest putting it into `search()` which will check if arabic or english is being written and change the component accordingly.